models/cont_delay_connection.h

```diff
@@ -141,8 +141,7 @@ class ContDelayConnection : public Connection< targetidentifierT >
   void set_status( const DictionaryDatum& d, ConnectorModel& cm );
 
   /**
-   * Check syn_spec dictionary for parameters that are not allowed for this
-   * connection. Will issue warning or throw error if a parameter is found.
+   * Checks to see if delay is given in syn_spec.
```

Comment:
This is a bit confusing. I would rather write `Throws exception if delays is given`. Otherwise, the reader may think that the check will throw and exception if the weight is missing.